packages/js/settings-editor/src/layout.tsx

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
Same with this file, move to `@woocommerce/admin-layout` because its the exact same as the product editor

Reply:
This is also very similar to CYS. 😄